### PR TITLE
Update bedrock-profile

### PR DESCRIPTION
**Pull request by JSAssassin**
Need to update bedrock-web-profile and bedrock-profile once
https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/72
digitalbazaar/bedrock-profile#39 are merged.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "bedrock-jsonld-document-loader": "^1.0.1",
     "bedrock-passport": "^6.0.0",
     "bedrock-permission": "^3.2.0",
-    "bedrock-profile": "^7.1.0",
+    "bedrock-profile": "github:digitalbazaar/bedrock-profile#issue-30",
```
Comment (aljones15) [inline package.json:56]: provided that this PR is still needed, can we swap the branches out for releases of `bedrock-profile`? If we can this should be good to go.

File: test/package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "bedrock-package-manager": "^1.0.1",
     "bedrock-passport": "^6.0.0",
     "bedrock-permission": "^3.2.0",
-    "bedrock-profile": "^7.1.0",
+    "bedrock-profile": "github:digitalbazaar/bedrock-profile#issue-30",
```
Comment (aljones15) [inline test/package.json:30]: same issue here. We need a release not a branch in package.json.

File: test/package.json [hunk 3]
```diff
@@ -36,7 +36,7 @@
     "bedrock-vc-issuer": "file:..",
     "bedrock-vc-revocation-list-context": "^1.0.0",
     "bedrock-veres-one-context": "^10.0.1",
-    "bedrock-web-profile-manager": "^6.2.1",
+    "bedrock-web-profile-manager": "github:digitalbazaar/bedrock-web-profile-manager#update-remove-api",
```
Comment (aljones15) [inline test/package.json:39]: has a release been made with this?

**Review by aljones15 — CHANGES_REQUESTED** [review]

File: test/mocha/20-recovery.js [hunk 4]
```diff
@@ -26,6 +26,7 @@ describe('Failure recovery', function() {
     agents = await helpers.insertIssuerAgent({
       id: accountId,
       token: 'token-43f47a1f-acaf-4dd1-8597-001a8b0637e3',
+      didMethod: 'key',
```
Comment (aljones15) [inline test/mocha/20-recovery.js:29]: can we not recover using `didMethod: 'v1'`?

File: test/package.json (hunk 2 above)
Comment (JSAssassin) [reply test/package.json]: Updated 2dd6e9e

File: test/package.json (hunk 3 above)
Comment (JSAssassin) [reply test/package.json]: yes. Updated here 2dd6e9e

File: package.json (hunk 1 above)
Comment (JSAssassin) [reply package.json]: Updated here 2dd6e9e

File: test/mocha/20-recovery.js (hunk 4 above)
Comment (JSAssassin) [reply test/mocha/20-recovery.js:30]: added a test here 699aa4b

**Review by aljones15 — APPROVED** [review]
wow, this is great. Thanks.

**Review by dlongley — APPROVED** [review]
Thanks!

**Pull request merged**